packages/engine/src/simulation/status.rs

```diff
@@ -23,6 +23,21 @@ pub struct SimStatus {
 }
 
 impl SimStatus {
+    /// Default value for `SimStatus` but not exposed to the user.
+    fn default() -> Self {
```

Comment:
It's the latter: I don't want to create `SimStatus::default()` because the `SimulationId` would be `0` then which could lead to bugs if not setting them afterward. By having a private `default()` method we only have to take care in the implementation, that `sim_id` is set correctly.

Reply:
But why do we have a default method at all is what I'm saying? Private or otherwise? Why not something like `new(id)` or a different method?

Reply:
Because this would be way too easy! 🙃 
Thanks for the suggestion, somehow I didn't think of that solution... 9ebeeca7da46a1f0940ef9b6bfd4061b69eb1627